### PR TITLE
Env file

### PR DESCRIPTION
**Pull request by ValdarRudman**

### CODE REVIEW

File: src/config/typeorm.config.ts [hunk 1]
```diff
@@ -16,13 +16,27 @@
...
+    entities: [__dirname + '/../**/*.entity.{js,ts}'],
+    synchronize: true,
+  }
+});
+
 /**
  * Config for database connetion.
  */
 export const typeOrmConfig: TypeOrmModuleOptions = {
   type: 'postgres',
```
Comment (SeanCondon) [inline src/config/typeorm.config.ts:37]: should these be referring the "config" values?

**Review by SeanCondon — APPROVED** [review]
A minor comment inline

File: src/config/typeorm.config.ts (hunk 1 above)
Comment (ValdarRudman) [reply src/config/typeorm.config.ts:37]: No. I was following the documentation to implement through this file but it does not seem to work. So I implemented it another way, and have left this as I want to take another look in the future.

Comment (SeanCondon) [reply src/config/typeorm.config.ts:37]: ok - thanks

**Comment by ValdarRudman** [comment]
The config files are not part of everything at the moment. There is a config service where you get the values from the env file. I will look into using config files instead of using the config service directly. There is nothing wrong with using it directly, but doing it through the config files will make it clearer.
I have tried to do this but was not working as intended.

**Pull request merged**